modules/isc_dhcpd/isc_dhcpd_test.go

```diff
@@ -157,3 +185,73 @@ func ensureCollectedHasAllChartsDimsVarsIDs(t *testing.T, dhcpd *DHCPd, collecte
 		}
 	}
 }
+
+func leaseDoesNotExist() *DHCPd {
```

Comment:
call those funtions `prepareDHCPd...` where `...` is the test case.

and it is not `lease` it is `LeaseDB`

> dhcpd.leases - DHCP client lease database